backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2154,3 +2154,23 @@ databaseChangeLog:
                   replaceIfExists: true
                   fullDefinition: false
                   selectQuery: SELECT internal_id, created_at, updated_at, last_seen, is_deleted FROM ${database.defaultSchemaName}.api_user
+  - changeSet:
+      id: add-patient-preferences-to-person-view
+      author: nathan@skylight.digital
+      comment: Add person patient preferences to person_no_phi_view
+      changes:
+        - createView:
+            viewName: person_no_phi_view
+            remarks: A subset of the person table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT p.internal_id, p.created_at, p.created_by, p.updated_at, p.updated_by, p.is_deleted, p.organization_id, p.facility_id, p.race, p.gender, p.ethnicity, p.county, p.state, p.employed_in_healthcare, p.role, p.resident_congregate_setting, p.tribal_affiliation, get_census_dob_group(p.birth_date) AS census_dob_group, pp.preferred_language, pp.test_result_delivery FROM ${database.defaultSchemaName}.person p JOIN patient_preferences pp on pp.internal_id=p.internal_id
+            rollback:
+              - dropView:
```

Comment:
If we're doing a replaceIfExists, do we actually need to drop it first?

But more to the point, I think we can actually specify the rollback as "re-run previous changeset `add-census-dob-group-column-to-person-view`" with a little tickling, which seems better from a DRY standpoint—see the third example on the [rollback tag description](https://docs.liquibase.com/concepts/basic/changeset.html#rollback-tag).

Reply:
@benwarfield-usds we do actually, since we're dropping a column - Postgres won't let you do that in a `REPLACE` as far as I could tell.

Reply:
Since we're not set up to easily test rollbacks locally, and I've spent a decent amount of time trying to rectify that without success, I would lean towards hewing as closely to vanilla SQL as possible for this work.

We should _definitely_ be leveraging Liquibase's Gradle plugin, but I'm not sure this ticket should bear the load of setting that up given that it doesn't seem trivial to do.

I've manually tested the SQL for the rollback and it LGTM. I'm happy with that if @benwarfield-usds is.

Reply:
Sorry I ghosted here, I just realized that my problem is that I switched work phones and stopped getting alerts. 🤦 

I had to catch up on the docs and the bug that Josh linked up, but I think there's a bigger problem here, which somewhat suggests that we should make this not able to be rolled back at all: if I'm right about interpreting the JIRA about `CREATE OR REPLACE`, we'd break metabase on a rollback.

Reply:
Great catch @nathancrtr! Having broken rollbacks is not great, and we might want to investigate whether we can fix them retroactively without breaking the changelog, but it's great to be able to prove that a properly nested rollback does what we think it will.